application/models/tag.py

```diff
@@ -1,11 +1,21 @@
 """ Access to Tag object from Neo4J """
 from neomodel import StructuredNode, StringProperty, RelationshipFrom
+
 from application.models.mixins import DefaultPropertyMixin, DefaultHelperMixin
 
 
-class Tag(StructuredNode, DefaultPropertyMixin, DefaultHelperMixin):  # pylint:disable=abstract-method
+# pylint:disable=abstract-method
+class Tag(StructuredNode, DefaultPropertyMixin, DefaultHelperMixin):
     """ The class that manages Tags """
     # schema = ArtifactSchema does not exist yet
```

Comment:
oh there is a lost comment here. Feel free to remove :D

Reply:
# schema = ArtifactSchema does not exist yet
 this one you meant right? will remove it.